qa/workunits/rbd/rbd_benchmark_clay_vs_jerasure.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+# This script benchmarks RBD image performance using Jerasure vs CLAY EC pools.
+# It runs both rbd bench and fio tests (kernel-mapped block device + rados engine)
+
+set -ex
+
+# === CONFIGURABLE PARAMETERS ===
+JERASURE_PROFILE="jerasure-ec-profile"
+CLAY_PROFILE="clay-ec-profile"
+
+# Pools
+JERASURE_DATA_POOL="jerasure_ec_pool"
+JERASURE_META_POOL="jerasure_meta"
+CLAY_DATA_POOL="clay_ec_pool"
+CLAY_META_POOL="clay_meta"
+
+# Images
+JERASURE_IMG="test_img_j"
+CLAY_IMG="test_img_c"
+IMG_SIZE="10G"
+
+# Benchmark params
+IO_TYPE="write"
+IO_SIZE="1M"
+IO_THREADS=4
+IO_TOTAL="4G"
+PATTERN="seq"
```

Comment:
I'm curious why 1M sequential writes were chosen for the benchmark -- do you expect it to be a common workload?

Reply:
I can't recommend anything because I'm struggling to see the purpose of this benchmark.  My understanding is that CLAY is a layered construction and as such it can't be expected to be universally faster than the plugin that is used underneath as the layer/building block (in this case Jerasure).  AFAIU the selling point of CLAY is savings in the amount of disk I/O and cluster network bandwidth that are required when an OSD is being recovered after a failure, not better IOPS in a steady state.  This script doesn't involve OSD recovery at all so I'm not sure I understand what performance differences you are trying to highlight here.